packages/helpers/scripts/ast_grep.js

```diff
@@ -87,7 +87,7 @@ export function ast_grep() {
                     source.prependLeft(range.start.index, `exports._ = exports.${func_name} = `);
                 });
 
-            const export_shortname = `export { ${func_name} as _}`;
+            const export_shortname = `export { ${func_name} as _ }`;
```

Comment:
this seemed to need a space, lmk if it is intended - no error thrown yet.

Reply:
This does not matter since ast-grep works on the AST, not based on string matching. 
And inserting a space enhances the visual formatting.

Reply:
Ah, understood.

Reply:
In fact, `export_alias` and the modified `match` have matched the same sentence. 
So I think we can omit one query.